src/orange/orangeinp/IntersectRegion.hh

```diff
@@ -424,7 +538,7 @@ class Involute final : public IntersectRegionInterface
  *   - \c theta polar angle of the shape's main axis, e.g. the segment defined
  *     by the centers of the Z faces. Validity range is `[0, 1/4)`;
  *   - \c phi azimuthal angle of the shape's main axis (as explained above).
- *     Validity range is `[0, 1)`.
+ *     Validity range is `[0, 1)`.
```

Comment:
I have no explanation for this whitespace change, which should be a no-op.

Reply:
weeeird

Reply:
Maybe Vidor committed some weird non-space space :\?

Reply:
Yeah there's some weird garbage in there, `0xC2A0`... no worries!